packages/subgraph/src/entities/TokenAllowance.ts

```diff
@@ -6,8 +6,12 @@ import { TokenAllowance } from "../../generated/schema";
 
 import { getUser } from "./User";
 
-export function getTokenAllowance(_token: Address, _owner: Address, _spender: Address): TokenAllowance {
-  let id = _token.toHexString() + _owner.toHexString() + _spender.toHexString();
+export function getTokenAllowance(
+  _token: Address,
+  _owner: Address,
+  _spender: Address
+): TokenAllowance {
+  let id = _token.toHexString() + "-" + _owner.toHexString() + "-" + _spender.toHexString();
```

Comment:
Should we set up a linter so we avoid these kind of changes?

Reply:
I meant the line above, but this is resolved with the comment about prettier config file below too, thanks!